proteus/mprans/GN_SW2DCV.h

```diff
@@ -83,7 +84,7 @@ inline double ENTROPY_FLUX2(const double &g, const double &h, const double &hu,
           g * h * z) *
          hv * one_over_hReg;
 }
-// FOR ESTIMATING MAX WAVE SPEEDS, GIVE REFERENCE
+// FOR ESTIMATING MAX WAVE SPEEDS
```

Comment:
I'm guessing @manuel-quezada was intending to come back at put a reference to the paper that derives these estimates.